nncf/tensorflow/tf_internals.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+ # pylint: disable=unused-import
+
+from packaging import version
+
+from nncf.tensorflow import tensorflow_version
+
+from tensorflow.python.eager import context as eager_context
+
+
+if version.parse(tensorflow_version) < version.parse('2.6'):
+    from tensorflow.python.keras import engine as keras_engine
+    from tensorflow.python.keras import Sequential, backend, layers
```

Comment:
Could you please explain why the `Sequential` was added here? It is a [public API](https://www.tensorflow.org/versions/r2.8/api_docs/python/tf/keras/Sequential) (`tf.keras.Sequential`).

Reply:
Right, I made it for compatibility which is mostly importing from `tensorflow.python.keras` in our current develop code but current code is importing from `tf.keras.Sequential` so it's not necessary.

Reply:
I added it for compatibility which is mostly using `tf.python.keras` API in develop code, but our current code is not using `Sequential` from `tf.python.keras` so actually no need to be changed.